compiler/rustc_lint/src/builtin.rs

```diff
@@ -305,9 +307,34 @@ declare_lint! {
     ///
     /// This lint is intended to restrict the usage of `unsafe`, which can be
     /// difficult to use correctly.
+    ///
+    /// ### Example
+    ///
+    /// ```rust,compile_fail
+    /// #![deny(unsafe_code)]
+    ///
+    /// #[no_mangle]
+    /// fn func0(){
+    ///
+    /// }
+    ///
+    /// fn main() {
+    ///     unsafe {
+    ///
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// {{produces}}
+    ///
+    /// ### Explanation
+    ///
+    /// This lint is intended to restrict the usage of `unsafe`,
+    /// and `no_mangle`, which can be
+    /// difficult to use correctly.
```

Comment:
this is the same lint; you need to expand the example and the explanation, not duplicate

Reply:
Thank you, I updated. :blue_heart: 

@rustbot review